Week-2/W2E1-index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <title>Webpage mockup</title>
+  <!--Google fonts-->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Ubuntu:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap"
+    rel="stylesheet">
+  <!-- CSS Stylesheet link -->
+  <link rel="stylesheet" href="W2E1-css/styles.css">
+  <!-- Font awesome link -->
+  <script src="https://kit.fontawesome.com/ae518c824b.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <header>
+    <img src="W2E1-images/blogger-logo.jpeg" alt="blog logo">
+  </header>
+  <div class="disc">
+    <h1>Stories that speak for themselves.</h1>
+  </div>
+  <div class="articles">
+    <div class="art art1">
+      <h2>History of Byte.</h2>
+      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>
+      <button type="button" name="button"><a href="#" type="Submit">Read more</a></button>
+    </div>
+    <div class="art art2">
+      <h2>Life of musician.</h2>
+      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>
+      <button type="button" name="button"><a href="#" type="Submit">Read more</a></button>
+    </div>
+    <div class="art art3">
+      <h2>Artist's home.</h2>
+      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. </p>
+      <button type="button" name="button"><a href="#" type="Submit">Read more</a></button>
+    </div>
+  </div>
```

Comment:
<form class="forms" action="index.html" method="post">
    <h1>Contact me</h1>
    <fieldset>
      <legend>Personal information:</legend>
      <label for="fname">First name:</label><br>
      <input type="text" id="fname" name="fname"><br>
      <label for="lname">Last name:</label><br>
      <input type="text" id="lname" name="lname"><br>
      <label for="email">Email id:</label><br>
      <input type="email" id="email" name="email"><br>
    </fieldset>
    <fieldset>
      <legend>Create your area here:</legend>
      <label for="login">Username:</label><br>
      <input type="text" id="login" name="login"><br>
      <label for="Cpass">Create password:</label><br>
      <input type="password" id="cpass" name="cpass"><br>
      <label for="Rpass">Retype password:</label><br>
      <input type="text" id="rpass" name="rpass"><br>
    </fieldset>
    <fieldset>
      <legend>Talk to us:</legend>
      <label for="message">Enter a message:</label><br>
      <textarea id="message" name="message" rows="10" cols="30"></textarea><br>
    </fieldset>
  <button type="submit" name="formsubmit"><a href="#">Submit</a></button>
  </form>